code/modules/projectiles/guns/manufacturer/frontier_import/ballistics.dm

```diff
@@ -0,0 +1,229 @@
+	mob_overlay_icon = 'icons/obj/guns/manufacturer/frontier_import/onmob.dmi'
+
+	icon_state = "pounder"
+	item_state = "pounder"
+	mag_type = /obj/item/ammo_box/magazine/c22lr_pounder_pan
+	burst_size = 1
+	fire_delay = 0.05 SECONDS
+	spread = 25
+	spread_unwielded = 50
+
+	fire_sound = 'sound/weapons/gun/smg/pounder.ogg'
+	rack_sound = 'sound/weapons/gun/smg/pounder_cocked.ogg'
+	rack_sound_vary = FALSE
+
+	load_sound_vary = FALSE
+	eject_sound_vary = FALSE
+	load_sound = 'sound/weapons/gun/smg/pounder_reload.ogg'
+	load_empty_sound = 'sound/weapons/gun/smg/pounder_reload.ogg'
+	eject_sound = 'sound/weapons/gun/smg/pounder_unload.ogg'
+	eject_empty_sound = 'sound/weapons/gun/smg/pounder_unload.ogg'
+
+	gun_firemodes = list(FIREMODE_FULLAUTO)
+	default_firemode = FIREMODE_FULLAUTO
+
+	manufacturer = MANUFACTURER_IMPORT
+	wield_slowdown = 0.5
+
+/obj/item/ammo_box/magazine/c22lr_pounder_pan
+	name = "pan magazine (.22 LR)"
+	desc = "A 50-round pan magazine for the Pounder machine gun. It's rather tiny, all things considered; it looks like it wouldn't pierce a single piece of armor."
+	icon_state = "firestorm_pan"
+	base_icon_state = "firestorm_pan"
+	ammo_type = /obj/item/ammo_casing/c22lr
+	caliber = "22lr"
+	max_ammo = 50
+	w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/ammo_box/magazine/c22lr_pounder_pan/update_icon_state()
+	. = ..()
+	icon_state = "firestorm_pan"
+
+/obj/item/gun/ballistic/automatic/hmg/shredder
+	name = "\improper Shredder"
+	desc = "A peculiar Frontiersman heavy machine gun. The tripod has been removed, a chainsaw grip for hip fire at the front has been added, and it has been rechambered to fire shotgun shells. Chambered in 12g."
+	icon = 'icons/obj/guns/manufacturer/frontier_import/48x32.dmi'
+	lefthand_file = 'icons/obj/guns/manufacturer/frontier_import/lefthand.dmi'
+	righthand_file = 'icons/obj/guns/manufacturer/frontier_import/righthand.dmi'
+	mob_overlay_icon = 'icons/obj/guns/manufacturer/frontier_import/onmob.dmi'
+
+	icon_state = "shredder"
+	item_state = "shredder"
+	mag_type = /obj/item/ammo_box/magazine/m12_shredder
+	spread = 15
+	recoil = 2
+	recoil_unwielded = 7
+	fire_delay = 0.16 SECONDS
+	mag_display_ammo = TRUE
+
+	bolt_type = BOLT_TYPE_STANDARD
+	show_magazine_on_sprite = TRUE
+	show_magazine_on_sprite_ammo = TRUE
+	tac_reloads = FALSE
+	fire_sound = 'sound/weapons/gun/hmg/shredder.ogg'
+	rack_sound = 'sound/weapons/gun/hmg/shredder_cocked_alt.ogg'
+
+	load_sound_vary = FALSE
+	eject_sound_vary = FALSE
+	load_sound = 'sound/weapons/gun/hmg/shredder_reload.ogg'
+	load_empty_sound = 'sound/weapons/gun/hmg/shredder_reload.ogg'
+	eject_sound = 'sound/weapons/gun/hmg/shredder_unload.ogg'
+	eject_empty_sound = 'sound/weapons/gun/hmg/shredder_unload.ogg'
+
+	manufacturer = MANUFACTURER_IMPORT
+	has_bipod = FALSE
+
+/obj/item/ammo_box/magazine/m12_shredder
+	name = "box magazine (12g)"
+	desc = "A 40-round box magazine for the Shredder heavy machine gun."
```

Comment:
name = "belt box (12g)"
desc = "A 40-round belt box for the Shredder heavy machine gun."